libweasyl/libweasyl/alembic/versions/40c00abab5f9_add_artist_tags.py

```diff
@@ -42,6 +42,7 @@ def upgrade():
                     )
     op.create_index('ind_artist_optout_tags_tagid', 'artist_optout_tags', ['tagid'])
     op.create_index('ind_artist_optout_tags_targetid', 'artist_optout_tags', ['targetid'])
+    op.execute('CREATE EXTENSION IF NOT EXISTS FUZZYSTRMATCH')
 
 
 def downgrade():
```

Comment:
Companion to the ``upgrade()`` block's additional line for this commit (the original state of the DB should be restored on execution of ``downgrade()``); add the following line at the end of the downgrade function:

``op.execute('DROP EXTENSION IF EXISTS FUZZYSTRMATCH')``